modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/EditAPIApplicationSchema.testcase

```diff
@@ -0,0 +1,216 @@
+@component-name = "portal-headless"
+definition {
+
+	property custom.properties = "feature.flag.LPS-178642=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Object";
+	property testray.main.component.name = "Headless Builder";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given New API Application created") {
+			ApplicationAPI.createAPIApplication(
+				baseURL = "my-app",
+				title = "My-app");
+		}
+
+		task ("And Given I go to the Schemas tab of API Application edition") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Schemas",
+				title = "My-app");
+		}
+	}
+
+	tearDown {
+		ApplicationAPI.deleteAllAPIApplication();
+
+		JSONObject.deleteAllCustomObjects(pageSize = 21);
```

Comment:
We can remove the `deleteAllCustomObjects` since there is no object definitions created in this test file.

Reply:
hi @pedroamarall , please help remove the unused
`JSONObject.deleteAllCustomObjects(pageSize = 21);`